app/models/data-operation.ts

```diff
@@ -175,6 +203,25 @@ export interface SRS {
   epsg?: string;
 }
 
+/**
+ * Transform a related URL from CMR to a Harmony related URL that can be used in the DataOperation.
+ * @param relatedUrl - A CmrRelatedUrl
+ * @returns the equivalent HarmonyRelatedUrl 
+ */
+export function cmrRelatedUrlToHarmony(relatedUrl: CmrRelatedUrl): HarmonyRelatedUrl {
+  const harmonyRelatedUrl: HarmonyRelatedUrl = {
```

Comment:
Let's say there's a future where we just want to pass on the UMM-Var object in its entirety.  Are we making it harder for ourselves by lower-camel-casing all the fields?  I'm kind of torn on whether we should just pass the object as we receive it vs trying to interpret it.

(Also, :shaking-angry-fist: at whomever decided Var fields should be upper-camel-case)

Reply:
Interesting point. I was only thinking of the current conventions of the JSON schema and the service-lib, and how not following those conventions might confuse things downstream.

Reply:
Got rid of this function.